companyweb_payment_info/wizards/payment_info_wizard.py

```diff
@@ -0,0 +1,396 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import datetime
+import html
+import re
+
+import zeep
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+import odoo.addons.companyweb_base.models.res_partner as cweb_partner
+
+CWEB_INVOICE_MAX_LEN = 4999
+VAT_RE = re.compile(r"^(BE)?\s?0?[0-9]{4}\s?\.?[0-9]{3}\s?\.?[0-9]{3}$")
+
+
+def _chunks(lst, chunk_size):
+    """Yield successive n-sized chunks from lst."""
+    for i in range(0, len(lst), chunk_size):
+        yield lst[i : i + chunk_size]
+
+
+class CompanyWebPaymentInfoWizard(models.TransientModel):
+
+    _name = "companyweb_payment_info.payment_info_wizard"
+    _description = "Companyweb Payment Info"
+    wizard_text = fields.Html("wizard_text")
+    wizard_step = fields.Char(default="step1")
+    wizard_email = fields.Char("wizard_email")
+
+    def payment_info_entry_point(self):
+        if self.wizard_step == "step1":
+            return self._cweb_payment_info_step1()
+        elif self.wizard_step == "step2":
+            return self._cweb_payment_info_step2()
+        elif self.wizard_step == "step3":
+            return True
+        else:
+            raise SystemError(_("Unexpected step %s") % self.wizard_step)
+
+    def _check_group(self):
+        if not self.env.user.has_group("companyweb_payment_info.cweb_upload"):
+            raise UserError(
+                _(
+                    """Companyweb : You need to be in the security group :
+                    Upload Companyweb Invoices to perform this action"""
+                )
+            )
+
+    @api.model
+    def _cweb_payment_info_step1(self):
+
+        self._check_group()
+
+        supplierVat = self.env.user.company_id.vat
+        if not supplierVat:
+            raise UserError(
+                _(
+                    "Companyweb : You need to set the vat field for the current company : %s"
+                    % self.env.user.company_id.name,
+                )
+            )
```

Comment:
Should we also check that the VAT number starts with BE ?

Reply:
done